crates/egui/src/widgets/text_edit/builder.rs

```diff
@@ -76,6 +76,7 @@ pub struct TextEdit<'t> {
     desired_height_rows: usize,
     event_filter: EventFilter,
     cursor_at_end: bool,
+    blink: bool,
```

Comment:
@emilk

I am not sure I fully understand your intention, so let me explain this.

TextEdit::blink stores the on/off setting for blinking. Even if there are multiple TextEdit instances in the program, the text cursor only exists in one that has focus, so I think the current configuration is appropriate.

paint_cursor() returns whether the current text cursor has been drawn or not. This is intended to call request_repaint(), but currently, request_repaint() does not work properly when the program does not have focus, so it is for the future and is a common configuration.

Therefore, the following code may or may not exist, and the current result will be the same:
```rust
    if is_drawn {
        ui.ctx().request_repaint();
    }
```

Please refer to this Pull Request and modify and apply it as emilk sees fit.

Reply:
I'm questioning why someone would want a blinking cursor in one TextEdit and not in another. I'd imagine a user would either want a blinking text cursor in all their TextEdits or in none of them, i.e. that the setting to turn on/off a blinking text cursor would belong in `egui::Visuals` instead of as individual options to `TextEdit`